build.gradle

```diff
@@ -465,6 +465,23 @@ configure(rootProject) {
 		doFirst {
 			classpath = files(subprojects.collect { it.sourceSets.main.compileClasspath })
 		}
+		doLast {
+			copy {
+				from("$buildDir/docs") {
+					filter { String line ->
+						line.startsWith("<head>") ? line.replace("<head>", "<head><link rel=\"icon\" type=\"image/png\" href=http://junit.org/junit5/assets/img/junit5-logo.png>") :
```

Comment:
Oops. It also worked without them and I used a different browser. I'll add them.